server/svix-server/src/db/models/eventtype.rs

```diff
@@ -101,9 +101,26 @@ pub fn schema_example() -> serde_json::Value {
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Default)]
-pub struct Schema(HashMap<String, Json>);
+pub struct Schema(pub HashMap<String, Json>);
 json_wrapper!(Schema);
 
+impl Schema {
+    // Return the first example found for a particular schema,
+    // if one exists:
+    pub fn example(&self) -> Option<&serde_json::Value> {
+        self.0
+            .get("1")
```

Comment:
Huh. Is this one of those situations where we're decoding an array into a map so the keys are numbered?

Reply:
We have a very strange convention at the moment where a schema must be stored with a top-level key of "1". I don't claim to understand it, but our existing APIs don't work if this is not specified when creating schemas/event types. A problem for another day, though, I suppose.

Reply:
yeah, okay